scipy/_lib/_array_api.py

```diff
@@ -428,18 +428,17 @@ def get_xp_devices(xp: ModuleType) -> list[str] | list[None]:
 
     # given namespace is not known to have a list of available devices;
     # return `[None]` so that one can use this in tests for `device=None`.
-    return [None] 
+    return [None]
 
 
-def scipy_namespace_for(xp: ModuleType) -> ModuleType:
-    """
-    Return the `scipy` namespace for alternative backends, where it exists,
-    such as `cupyx.scipy` and `jax.scipy`. Useful for ad hoc dispatching.
+def scipylike_namespace_for(xp: ModuleType) -> ModuleType:
+    """Return the `scipy`-like namespace of a non-NumPy backend
 
-    Default: return `scipy` (this package).
+    That is, return the namespace corresponding with backend `xp` that contains
+    `scipy` sub-namespaces like `linalg` and `special`. If no such namespace exists,
+    return None. Useful for dispatching.
```

Comment:
nit

```suggestion
    return `None`. Useful for dispatching.
```

Reply:
Unfortunately these don't link with intersphinx at this time. In the meantime, I'll use double.

Reply:
The function isn't public API anyway so not sure that matters - was just a suggestion for readability. Clearly not a big deal though, hence just a nit.